cicd/github-actions-oidc/README.md

```diff
@@ -0,0 +1,16 @@
+This templates show an example of how to authenticate against AWS using Open ID Connect (OIDC) and deploy/destroy a stack
+
+For more info on how to configure your AWS account, read [this blog post](https://benoitboure.com/securely-access-your-aws-resources-from-github-actions).
```

Comment:
I was suggesting adding your awesome tutorial then I found it here 🎉  Glad that you have included it already 👌🏻